docs/tutorial/part-one/index.md

```diff
@@ -95,96 +95,96 @@ export default () => (
 )
 ```
 
-![Add image](04-add-image.png)
+![Thêm hình ảnh](04-add-image.png)
 
-### Wait… HTML in our JavaScript?
+### Đợi đã... HTML trong JavaScript?
 
-_If you’re familiar with React and JSX, feel free to skip this section._ If you haven’t worked with the React framework before, you may be wondering what HTML is doing in a JavaScript function. Or why we’re importing `react` on the first line but seemingly not using it anywhere. This hybrid “HTML-in-JS” is actually a syntax extension of JavaScript, for React, called JSX. You can follow along with this tutorial without prior experience with React, but if you’re curious, here’s a brief primer…
+_Nếu như bạn quen thuộc với React và JSX, bạn có thể thoải mái bỏ qua phần này._ Nếu như bạn chưa từng làm việc với bộ khung (framework) React trước đây, bạn có thể tự hỏi HTML làm gì trong một hàm JavaScript. Hoặc là tại sao chúng ta nhập khẩu `react` ở dòng đầu nhưng dường như không sử dụng nó ở bất cứ đâu. Phiên bản lai "HTML trong JS" này trên thực tế là một cú pháp mở rộng của JavaScript, đối với React, gọi là JSX. Bạn có thể làm theo bài hướng dẫn này mà không cần kinh nghiệm với React, nhưng nếu bạn tò mò, đây là một đoạn trích ngắn gọn…
 
-Consider the original contents of the `src/pages/index.js` file:
+Xem xét nội dung ban đầu của tệp `src/pages/index.js`:
 
 ```jsx:title=src/pages/index.js
 import React from "react"
 
 export default () => <div>Hello world!</div>
 ```
 
-In pure JavaScript, it looks more like this:
+Trong JavaScript thuần, nó trông như thế này:
 
 ```javascript:title=src/pages/index.js
 import React from "react"
 
 export default () => React.createElement("div", null, "Hello world!")
 ```
 
-Now you can spot the use of the `'react'` import! But wait. You’re writing JSX, not pure HTML and JavaScript. How does the browser read that? The short answer: It doesn’t. Gatsby sites come with tooling already set up to convert your source code into something that browsers can interpret.
+Bây giờ bạn có thể phát hiện ra việc sử dụng nhập khẩu `'react'`! Nhưng chờ đã, Bạn đang viết JSX, không phải thuần HTML và JavaScript. Làm cách nào mà trình duyệt đọc được? Câu trả lời ngắn ngọn là: Nó không có. các trang Gatsby đi kèm với công cụ đã được thiết lập sẵn để chuyển đổi mã nguồn của bạn thành một cái gì đó mà trình duyệt có thể diễn giải.
```

Comment:
Bạn giữ import nguyên bản nhá
It doesn't -> Nó không đọc được
"các trang Gatsby..." -> Capitalize đầu câu giúp mình nhé